vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionLanguage.java

```diff
@@ -0,0 +1,93 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+@Entity
+public class ExhibitionLanguage {
+
+    public ExhibitionLanguage() {
+        super();
+    }
+
+    @Id
+    @GeneratedValue(generator = "exhibit_language_id_generator")
+    @GenericGenerator(name = "exhibit_language_id_generator", parameters = @Parameter(name = "prefix", value = "EXH"), strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
```

Comment:
`EXH` is the prefix for exhibitiion objects. here it should be a prefix for language objects.